### PR TITLE
feat(Style): Change baseline from 11px to 12px

### LINKED ISSUE #721: [v1] Change baseline from 11px to 12px
Currently `$iui-baseline` is set to 11px.  This causes issues when dividing baseline by 2 (5.5px).

### PR DESCRIPTION
**Pull request by FlyersPh9**
- Change `$iui-baseline: 11px` to `$iui-baseline: $iui-sm;` which is equal to 12px.
- Change all instances of `11px` to `12px` and `5.5px` to `6px` in demo page `<head>` and `demo.css`.
- Replace all instances of `round($iui-baseline * 0.5)` to `($iui-baseline * 0.5)`.
- Required [an update to `stepper.js` for test image to display correctly](https://github.com/iTwin/iTwinUI/pull/713/#issuecomment-1197188155).  Unsure as to why. 🤔

Closes #721.

### CODE REVIEW

File: packages/itwinui-css/src/radio-tile/radio-tile.scss [hunk 1]
```diff
@@ -124,7 +124,8 @@
   }
 
   @include focus-visible {
-    + * { // outline for checkmark
+    + * {
```
Comment (adamhock) [inline packages/itwinui-css/src/radio-tile/radio-tile.scss:127]: Pesky formatting strikes again 😈

File: packages/itwinui-css/backstop/tests/assets/demo.css [hunk 2]
```diff
@@ -61,12 +61,12 @@ body {
```
Comment (adamhock) [inline packages/itwinui-css/backstop/tests/assets/demo.css:61]: Is it okay to leave `margin: 11px 0` for `hr`

**Review by adamhock — COMMENTED** [review]
I did a quick repo search for 11px and saw a few instances in the htmls where the padding was `11px 12px`. Do these need to also be changed with this PR?

Also, there's a commented line in slider.scss about `$tick-height` being `11px`

File: packages/itwinui-css/src/radio-tile/radio-tile.scss (hunk 1 above)
Comment (FlyersPh9) [reply packages/itwinui-css/src/radio-tile/radio-tile.scss]: Dang, latest push should revert this.

File: packages/itwinui-css/backstop/tests/assets/demo.css (hunk 2 above)
Comment (FlyersPh9) [reply packages/itwinui-css/backstop/tests/assets/demo.css:61]: Updated to use 12px here and in a couple other places that were missed.

**Comment by bentleyvk** [comment]
Stepper has some background changes:
image

File: packages/itwinui-css/src/style/global-variables.scss [hunk 3]
```diff
@@ -8,16 +8,16 @@ $iui-border-radius: 4px;
...
+$iui-component-height-small: $iui-baseline * 3 - $iui-component-offset * 2; // 28px
+$iui-component-height: $iui-baseline * 4 - $iui-component-offset * 2; // 40px
+$iui-component-height-large: $iui-baseline * 5 - $iui-component-offset * 2; // 52px
 
 $iui-component-padding-horizontal: $iui-sm; // 12px
 
-$iui-component-padding-vertical: (($iui-component-height - $iui-line-height) * 0.5) - 1; // 12.5px
-$iui-component-padding-vertical-small: (($iui-component-height-small - $iui-line-height) * 0.5) - 1; // 7px
-$iui-component-padding-vertical-large: (($iui-component-height-large - $iui-line-height) * 0.5) - 1; // 18px
+$iui-component-padding-vertical-small: (($iui-component-height-small - $iui-line-height) * 0.5) - 1; // 1px
```
[the remark this answers is not in the document]
Comment (FlyersPh9) [reply packages/itwinui-css/src/style/global-variables.scss:19]: Unless I'm doing my math wrong, I think the values we had previously were wrong. 😬

## Old

```scss
$iui-component-padding-vertical-small: (($iui-component-height-small - $iui-line-height) * 0.5) - 1; // 7px
$iui-component-padding-vertical: (($iui-component-height - $iui-line-height) * 0.5) - 1; // 12.5px
$iui-component-padding-vertical-large: (($iui-component-height-large - $iui-line-height) * 0.5) - 1; // 18px
```
**Small:** ((27** - 22) * 0.5) - 1 = **1.5px**
**Medium:** ((38 - 22) * 0.5) - 1 = **7px**
**Large:** ((49 - 22) * 0.5) - 1 = **12.5px**


## New

```scss
$iui-component-padding-vertical-small: (($iui-component-height-small - $iui-line-height) * 0.5) - 1; // 1px
$iui-component-padding-vertical: (($iui-component-height - $iui-line-height) * 0.5) - 1; // 7px
$iui-component-padding-vertical-large: (($iui-component-height-large - $iui-line-height) * 0.5) - 1; // 13px
```
**Small:** ((28 - 24) * 0.5) - 1 = **1px**
**Medium:** ((40 - 24) * 0.5) - 1 = **7px**
**Large:** ((52 - 24) * 0.5) - 1 = **13px**

## Comparison

```diff
- $iui-component-padding-vertical-small: 1.5px;
+ $iui-component-padding-vertical-small: 1px;
$iui-component-padding-vertical: 7px;
- $iui-component-padding-vertical-large: 12.5px;
+ $iui-component-padding-vertical-large: 13px;
```

[the remark this answers is not in the document]
Comment (FlyersPh9) [reply packages/itwinui-css/src/style/global-variables.scss:19]: Also confirmed in dev tools (in order of small, medium, large):

## Old
Screen Shot 2022-07-27 at 1 08 55 PM Screen Shot 2022-07-27 at 1 11 13 PM Screen Shot 2022-07-27 at 1 12 23 PM
 
## New
Screen Shot 2022-07-27 at 1 13 11 PM Screen Shot 2022-07-27 at 1 13 30 PM Screen Shot 2022-07-27 at 1 13 49 PM

File: packages/itwinui-css/backstop/tests/color.html [hunk 4]
```diff
@@ -31,7 +31,7 @@
         width: auto;
       }
       section[id^='demo'] > * {
-        padding: 11px 24px;
+        padding: 24px;
```
[the remark this answers is not in the document]
Comment (FlyersPh9) [reply packages/itwinui-css/backstop/tests/color.html]: Changed to `12px 24px` to more closely match the original.

File: packages/itwinui-css/src/color-picker/color-picker.scss [hunk 5]
```diff
@@ -147,13 +147,13 @@ $iui-hover-box-shadow: 0 0 1px $iui-xxs + 1 rgba(var(--iui-color-foreground-body
   position: relative;
```
[the remark this answers is not in the document]
Comment (FlyersPh9) [reply packages/itwinui-css/src/color-picker/color-picker.scss:147]: Reverted this change.

File: packages/itwinui-css/src/color-picker/color-picker.scss [hunk 6]
```diff
@@ -147,13 +147,13 @@ $iui-hover-box-shadow: 0 0 1px $iui-xxs + 1 rgba(var(--iui-color-foreground-body
...
   cursor: crosshair;
   width: 100%;
-  height: $iui-baseline * 19;
+  aspect-ratio: 1 / 1;
```
[the remark this answers is not in the document]
Comment (FlyersPh9) [reply packages/itwinui-css/src/color-picker/color-picker.scss]: Reverted this change.

**Comment by FlyersPh9** [comment]
> Stepper has some background changes: image

@bentleyvk that is very strange, this PR had no changes in any of the stepper files.  After making this change I was able to get the test image to appear as originally expected:

```diff
actions: [hover('#demo-default .iui-clickable:first-child')],
actions: [hover('#demo-default .iui-clickable:first-child .iui-stepper-circle')],
```

I'm going to go through the rest of the test images a little more closely after this.

File: packages/itwinui-css/src/style/global-variables.scss (hunk 3 above)
[the remark this answers is not in the document]
Comment (bentleyvk) [reply packages/itwinui-css/src/style/global-variables.scss:19]: lol

**Review by bentleyvk — APPROVED** [review]

**Review by adamhock — APPROVED** [review]

**Pull request merged**